.github/workflows/application-signals-e2e-test.yml

```diff
@@ -38,53 +38,180 @@ jobs:
+
+  ec2-default-v22-amd64:
+    needs: [ upload-main-build ]
+    uses: aws-observability/aws-application-signals-test-framework/.github/workflows/java-ec2-default-test.yml@main
+    secrets: inherit
+    with:
+      aws-region: us-east-1
+      caller-workflow-name: 'main-build'
+      java-version: '22'
+      cpu-architecture: 'x86_64'
+
+  #
+  # DOCKER DISTRIBUTION LANGUAGE VERSION COVERAGE
+  # DEFAULT SETTING: {Java Version}, EKS, AMD64, AL2
+  #
+
+  eks-v8-amd64:
+    uses: aws-observability/aws-application-signals-test-framework/.github/workflows/java-eks-test.yml@main
+    secrets: inherit
+    with:
+      aws-region: us-east-1
+      test-cluster-name: 'e2e-adot-test'
+      adot-image-name: ${{ inputs.adot-image-name }}
+      caller-workflow-name: 'main-build'
+      java-version: '8'
+
+  eks-v11-amd64:
+    needs: eks-v8-amd64
+    uses: aws-observability/aws-application-signals-test-framework/.github/workflows/java-eks-test.yml@main
+    secrets: inherit
+    with:
+      aws-region: us-east-1
+      test-cluster-name: 'e2e-adot-test'
+      adot-image-name: ${{ inputs.adot-image-name }}
+      caller-workflow-name: 'main-build'
+      java-version: '11'
+
+  eks-v17-amd64:
+    needs: eks-v11-amd64
+    uses: aws-observability/aws-application-signals-test-framework/.github/workflows/java-eks-test.yml@main
     secrets: inherit
     with:
       aws-region: us-east-1
+      test-cluster-name: 'e2e-adot-test'
+      adot-image-name: ${{ inputs.adot-image-name }}
+      caller-workflow-name: 'main-build'
+      java-version: '17'
+
+  eks-v21-amd64:
+    needs: eks-v17-amd64
+    uses: aws-observability/aws-application-signals-test-framework/.github/workflows/java-eks-test.yml@main
+    secrets: inherit
+    with:
+      aws-region: us-east-1
+      test-cluster-name: 'e2e-adot-test'
+      adot-image-name: ${{ inputs.adot-image-name }}
       caller-workflow-name: 'main-build'
+      java-version: '21'
 
-  java-eks-e2e-test:
+  eks-v22-amd64:
+    needs: eks-v21-amd64
     uses: aws-observability/aws-application-signals-test-framework/.github/workflows/java-eks-test.yml@main
     secrets: inherit
     with:
       aws-region: us-east-1
       test-cluster-name: 'e2e-adot-test'
       adot-image-name: ${{ inputs.adot-image-name }}
       caller-workflow-name: 'main-build'
+      java-version: '22'
 
-  java-metric-limiter-e2e-test:
-    needs: [ java-eks-e2e-test ]
+  #
+  # PACKAGED DISTRIBUTION PLATFORM COVERAGE
+  # DEFAULT SETTING: Java 11, {Platform}, AMD64, AL2
+  #
+
+  ec2-asg-v11-amd64:
+    needs: [ upload-main-build ]
+    uses: aws-observability/aws-application-signals-test-framework/.github/workflows/java-ec2-asg-test.yml@main
+    secrets: inherit
+    with:
+      aws-region: us-east-1
+      caller-workflow-name: 'main-build'
+      java-version: '11'
+
+  #
+  # DOCKER DISTRIBUTION PLATFORM COVERAGE
+  # DEFAULT SETTING: Java 11, {Platform}, AMD64, AL2
+  #
+
+  metric-limiter-v11-amd64:
```

Comment:
nit: Not technically accurate. This test is a specific functionality of the instrumentation, not a specific configuration. Just move this test to the bottom and label it as a functional test case

Reply:
Done